RecoLocalCalo/HcalRecAlgos/plugins/HcalRecAlgoESProducer.cc

```diff
@@ -94,5 +96,19 @@ HcalRecAlgoESProducer::produce(const HcalSeverityLevelComputerRcd& iRecord)
    return myComputer ;
 }
 
+void HcalRecAlgoESProducer::fillDescriptions(edm::ConfigurationDescriptions& descriptions) {
+    edm::ParameterSetDescription desc;
+    desc.add<uint32_t>("phase",0);
+    desc.add<std::vector<std::string>>("RecoveredRecHitBits");
+    desc.add<std::vector<std::string>>("DropChannelStatusBits");
+    descriptions.add("hcalRecAlgos",desc);
+
+    edm::ParameterSetDescription desc_sevlvl;
+    desc_sevlvl.add<uint32_t>("Level");
+    desc_sevlvl.add<std::vector<std::string>>("RecHitFlags");
+    desc_sevlvl.add<std::vector<std::string>>("ChannelStatus");
+    desc_sevlvl.addVPSet("SeverityLevels",desc_sevlvl);
+}
```

Comment:
Just skimming the conversation quickly it appears to me the issue has already been resolved, but just in case it helps here is a link to the documentation describing the exact syntax for VPSets in fillDescriptions functions.

https://twiki.cern.ch/twiki/bin/view/CMSPublic/SWGuideConfigurationValidationAndHelp#ParameterSets_that_Contain_Vecto